sentry-log/src/lib.rs

```diff
@@ -34,6 +32,6 @@ mod converters;
 mod integration;
 mod logger;
 
-pub use converters::{breadcrumb_from_record, event_from_record};
+pub use converters::*;
 pub use integration::LogIntegration;
-pub use logger::Logger;
+pub use logger::{NoopLogger, SentryLogger};
```

Comment:
Exports for some of the types are missing, namely `LevelFilter` and `RecordMapping`. There are no doc tests that check for this, either.